go.mod

```diff
@@ -263,5 +262,5 @@ replace (
 	github.com/cosmos/ledger-cosmos-go => github.com/cosmos/ledger-cosmos-go v0.12.4
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
-	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.35.0-tm-v0.34.29
+	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.35.0-tm-v0.34.29.0.20240416162540-f96cc8fa3ee6
```

Comment:
This needs to be updated to point to one of the major branches in core, wright now, it points to `sanaz/cutom-core-for-knuu-test` i.e., [#1309](https://github.com/celestiaorg/celestia-core/pull/1309)